nrsysmond.service

```diff
@@ -1,10 +1,13 @@
 [Unit]
 Description=New Relic Linux Server Monitor (nrsysmond)
-After=docker.service
-Requires=docker.service
+Requires=docker.socket
```

Comment:
@jlsalvador could you explain why `docker.socket` is better than `docker.service`?
Is the `After=network.target` really required?